.storybook/config.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { configure, addDecorator } from '@storybook/react';
+import { withKnobs } from '@storybook/addon-knobs';
+import { withNotes } from '@storybook/addon-notes';
+import { withInfo } from '@storybook/addon-info';
+import { checkA11y } from '@storybook/addon-a11y';
+import AppLayout from 'storybookHelpers/components/app-layout/app-layout';
+
+
+const req = require.context('../src/components', true, /\.stories\.js$/);
+
+function loadStories() {
+  req.keys().forEach(filename => req(filename));
+}
+
+addDecorator(withKnobs);
+addDecorator(withNotes);
+addDecorator(withInfo);
+addDecorator(checkA11y);
```

Comment:
As below, I think this may run the A11Y tests over all of the rendered elements, i.e. the table for the prop types. Because we only want to run checks on our own components, we should probably add this decorator at component level, rather than globally here.

Reply:
This decorator is for story not for component. Other addons don't affect it.

Reply:
Ok good, this behaviour seems to have changed since we looked at the prototype.